WcaOnRails/app/views/static_pages/faq.html.erb

```diff
@@ -51,6 +51,21 @@
         <%= link_to "Cubing USA's competitor tutorial", "http://www.cubingusa.com/ctutorial.php", target: "_blank" %>, which gives several nice hints for newcomers.
       <% end %>
 
+      <%= add_answer "How can I have a WCA Competition in my hometown?" do %>
+        <p>
+          If you are interested in organizing a competition in your hometown, then it’s highly recommended to attend at least one or two competitions as a competitor to learn from the experience.
```

Comment:
(Apologies if the wording isn't supposed to be reviewed)

I'd trim this to
`If you are interested in organizing a competition, it’s highly recommended to attend at least one or two competitions as a competitor to learn from the experience.